content/blog/2024-10-29-almalinux-endorsed-distro.md

```diff
@@ -0,0 +1,34 @@
+---
+
+Earlier today we announced that AlmaLinux has been selected to join the [Endorsed Distributions list on Microsoft Azure](https://learn.microsoft.com/en-us/azure/virtual-machines/linux/endorsed-distros), a unique distinction with only 6 other distributions. The press release is included below with all of the details. 
+
+We are so grateful to the incredible Azure team for this, and are so excited to see our community supported in this way. 
+
+==========
+
+**FORT MYERS, Fla. -- October 29, 2024** -- The [AlmaLinux OS Foundation](https://almalinux.org/), the nonprofit that stewards the free and community-governed open source enterprise Linux distribution, today announced that AlmaLinux has been selected to join the [Endorsed Distributions list on Microsoft Azure](https://learn.microsoft.com/en-us/azure/virtual-machines/linux/endorsed-distros), a unique distinction with only 6 other distributions.
+
+Starting today, AlmaLinux images featured on Azure will be considered an endorsed distribution. Through collaborative engineering engagement between the two organizations, the most up-to-date AlmaLinux versions are regularly made available as Azure images, backed by robust in-Azure update infrastructure and an ongoing focus on fostering a straightforward process for maintaining images.
+
+"On behalf of the entire AlmaLinux OS Foundation board of directors, we are thrilled to see AlmaLinux join the Endorsed Distributions list for Microsoft Azure," said benny Vasquez, chair of the AlmaLinux OS Foundation. "It's clear that Azure is committed to supporting distributions that are highly adopted and used within their incredible platform. With more than half of all Azure offerings based on Linux, it seems only natural for Azure to continue to expand its list of endorsed distributions -- and we're delighted to play such a role."
+
+"We are pleased to add AlmaLinux as an endorsed, CentOS-compatible Linux distribution in Azure, backed up by a close engineering and support relationship between our respective teams," said Microsoft Azure platform Chief Product Officer and Corporate Vice President of Azure Cloud Native Jeremy Winter. "This Azure-AlmaLinux collaboration underscores our ongoing efforts to deepen cooperation with key players in the open-source community to address Azure customers' desire for stable, well-supported solutions."
+
+AlmaLinux is available via Azure Marketplace for the [x64](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux-x86_64?tab=Overview), [Arm64](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux-arm) architectures, including Microsoft's latest Cobalt 100-based virtual machines, as well as a customized image for [HPC workloads](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux-hpc). Images are also available via Community Gallery and [Windows Subsystem for Linux (WSL)](https://wiki.almalinux.org/documentation/wsl).
```

Comment:
Since the Windows Subsystem for Linux is not a technology related/connected/based on/to the Azure, our WSL images are different from the VM images on Azure. Can we make a correction here?

They are both using the Hyper-V virtualization though.